src/integration-test/java/se/sundsvall/datawarehousereader/apptest/customer/GetCustomerDetailsIT.java

```diff
@@ -12,27 +12,37 @@
 /**
  * Get customer details tests
  * 
- * @see src/test/resources/db/scripts/testdata.sql for data setup.
+ * @see src/test/resources/db/scripts/initialize.sql for data setup. Uses the function "kundinfo.fnCustomerDetails".
  */
 @WireMockAppTestSuite(files = "classpath:/GetCustomerDetails/", classes = Application.class)
 class GetCustomerDetailsIT extends AbstractAppTest {
 
 	private static final String RESPONSE_FILE = "response.json";
 
 	@Test
-	void test01_getByPartyId() {
+	void test01_getWithOrgNoAndPartyIds() {
 		setupCall()
-			.withServicePath("/customer/details?partyId=70933c1c-2094-44e6-9abe-57b8a3c0ecd9&partyId=318b7ec8-aab8-41e2-82f5-796bdd5cebf2&partyId=10e69eb7-3cb4-442f-8cdb-2e998080dbb1")
+			.withServicePath("/customer/details?partyId=9f395f51-b5ed-401b-b700-ef70cbb15d79&partyId=9f395f51-b5ed-401b-b700-ef70cbb15d80&partyId=9f395f51-b5ed-401b-b700-ef70cbb15d81&customerEngagementOrgId=5564786647")
```

Comment:
Maybe skip one partyId to verify that you just get the 2 asked for (and not all 3, which is the same as just filtering on customerEngagementOrgid)?

Reply:
Good point, I'll fix!